spec/rakelib/support/cookies_spec.rb

```diff
@@ -10,19 +10,20 @@
   let(:session) { Session.create(uuid: user.uuid, token: 'abracadabra') }
 
   describe '#api_session_header' do
-    # much of this code comes from here: https://stackoverflow.com/a/51579296
+    # much of this code comes from here: https://gist.github.com/wildjcrt/6359713fa770d277927051fdeb30ebbf
     def decrypt_session_cookie(cookie)
-      salt = Rails.application.config.action_dispatch.authenticated_encrypted_cookie_salt
-      encrypted_cookie_cipher = 'aes-256-gcm'
+      config = Rails.application.config
+      salt   = config.action_dispatch.authenticated_encrypted_cookie_salt
+      encrypted_cookie_cipher = config.action_dispatch.encrypted_cookie_cipher || 'aes-256-gcm'
 
-      key_generator = ActiveSupport::KeyGenerator.new(Rails.application.secrets.secret_key_base, iterations: 1000)
+      key_generator = ActiveSupport::KeyGenerator.new(Rails.application.secret_key_base, iterations: 1000)
```

Comment:
Did this key just move?

Reply:
Apparently this is a Rails helper for finding the secret_key_base. 
 https://api.rubyonrails.org/classes/Rails/Application.html#method-i-secret_key_base

TLDR this method checks `ENV`, then the `credentials.yml`, and finally the old-style `secrets.yml`.  Byebug showed me these values were the same here